E721 do not compare types

```
Linux
/home/runner/work/monai-code-formatter/monai-code-formatter/src_dir/tests/test_selfattention.py:65:16: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
/home/runner/work/monai-code-formatter/monai-code-formatter/src_dir/tests/test_transformerblock.py:69:16: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
/home/runner/work/monai-code-formatter/monai-code-formatter/src_dir/tests/test_vit.py:163:16: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
3     E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
3
Check failed!
```

--- a/.github/workflows/codeql-analysis.yml
+++ b/.github/workflows/codeql-analysis.yml
@@ -64,6 +64,9 @@ jobs:
 
     - name: Build
       run: |
+        rm -rf /opt/hostedtoolcache/{node,go,Ruby,Java*}
+        ls -al /opt/hostedtoolcache
+        rm -rf /usr/share/dotnet/
         python -m pip install -U pip wheel
         python -m pip install -r requirements-dev.txt
         BUILD_MONAI=1 ./runtests.sh --build

--- a/tests/test_selfattention.py
+++ b/tests/test_selfattention.py
@@ -62,7 +62,7 @@ class TestResBlock(unittest.TestCase):
         # be  not able to access the matrix
         no_matrix_acess_blk = SABlock(hidden_size=hidden_size, num_heads=num_heads, dropout_rate=dropout_rate)
         no_matrix_acess_blk(torch.randn(input_shape))
-        assert type(no_matrix_acess_blk.att_mat) == torch.Tensor
+        assert isinstance(no_matrix_acess_blk.att_mat, torch.Tensor)
         # no of elements is zero
         assert no_matrix_acess_blk.att_mat.nelement() == 0
 

--- a/tests/test_transformerblock.py
+++ b/tests/test_transformerblock.py
@@ -66,7 +66,7 @@ class TestTransformerBlock(unittest.TestCase):
             hidden_size=hidden_size, mlp_dim=mlp_dim, num_heads=num_heads, dropout_rate=dropout_rate
         )
         no_matrix_acess_blk(torch.randn(input_shape))
-        assert type(no_matrix_acess_blk.attn.att_mat) == torch.Tensor
+        assert isinstance(no_matrix_acess_blk.attn.att_mat, torch.Tensor)
         # no of elements is zero
         assert no_matrix_acess_blk.attn.att_mat.nelement() == 0
 

--- a/tests/test_vit.py
+++ b/tests/test_vit.py
@@ -160,7 +160,7 @@ class TestViT(unittest.TestCase):
         # no data in the matrix
         no_matrix_acess_blk = ViT(in_channels=in_channels, img_size=img_size, patch_size=patch_size)
         no_matrix_acess_blk(torch.randn(in_shape))
-        assert type(no_matrix_acess_blk.blocks[0].attn.att_mat) == torch.Tensor
+        assert isinstance(no_matrix_acess_blk.blocks[0].attn.att_mat, torch.Tensor)
         # no of elements is zero
         assert no_matrix_acess_blk.blocks[0].attn.att_mat.nelement() == 0
 

